### DISCUSSION TITLE
Implementing i2c1 with a pico shield

### DISCUSSION THREAD

**Discussion by joeker64** [zephyrproject-rtos/zephyr#60318] [Q&A]
Hi all :),

I am adding a pico shield which uses i2c1 pins GP6&GP7. I was going to add support for i2c1 in the default rpi_pico device tree until I noticed this commit which removes the i2c1 node:
https://github.com/zephyrproject-rtos/zephyr/commit/9eb7ee6ed8308bcee88bc87b13a7c5991166e2bf

I am happy moving the pinctrl information into the shield overlay but when setting "pico_i2c" (not added yet) in the "rpi_pico-common.dtsi" it will be "pico_i2c: &i2c1 {}" but there will be no default i2c1 information. 

Would this be okay? :)

**Comment by ndrs-pst** [comment]
@joeker64, I think you can update this discussion as closed since you already bring back from this PR https://github.com/zephyrproject-rtos/zephyr/pull/60384 😄
